routers/api/v1/api.go

```diff
@@ -807,7 +807,9 @@ func RegisterRoutes(m *macaron.Macaron) {
 									Get(repo.GetPullReviewComments)
 							})
 						})
-
+						m.Combo("/requested_reviewers").
```

Comment:
The "object" we're acting on is a review_request - therefore this should be `/review_requests`.

```suggestion
						m.Combo("/review_requests").
```

Probably should also have a Get.

Reply:
To follow GH also:
https://developer.github.com/v3/pulls/review_requests/#request-reviewers-for-a-pull-request

Not add get handle, Because it has been considered as a type of review whch can be getted by get reviews api.

Reply:
Thanks for clarifying